cnf-certification-test/accesscontrol/suite.go

```diff
@@ -133,7 +133,7 @@ func TestSecConCapabilities(env *provider.TestEnvironment) {
 		}
 	}
 	tnf.ClaimFilePrintf("bad containers: %v", badContainers)
-	tnf.GomegaExpectSliceBeNil(badContainers)
+	env.GomegaExpectSliceBeNil(badContainers)
```

Comment:
one last comment, this should be ExpectSliceBeEmpty 
the slice badContainers is not nil.

Reply:
never mind. just saw the declaration and it's var badContainers []string not :=[]string{}